### PR TITLE
Playlist Fragment

### PR DESCRIPTION
**Pull request by CallMeAlasca**
Linked Task: #RASPIO-15

### CODE REVIEW

File: app/app/src/main/java/me/bickositieff/raspio/ui/models/Song.kt [hunk 1]
```diff
@@ -0,0 +1,15 @@
+package me.bickositieff.raspio.ui.models
+
+import androidx.annotation.DrawableRes
+
+class Song(
+    val title: String,
+    val artist: String,
+    val duration: Int,
+    @DrawableRes
+    val image: Int?
```
Comment (profiluefter) [inline app/app/src/main/java/me/bickositieff/raspio/ui/models/Song.kt:10]: Will probably not be usable like that but we'll change it when we know how we'll use it

File: app/app/src/main/java/me/bickositieff/raspio/ui/playlist/PlaylistFragment.kt [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+import android.view.ViewGroup
+import android.widget.TextView
+import androidx.fragment.app.Fragment
+import androidx.lifecycle.ViewModelProvider
+import androidx.recyclerview.widget.RecyclerView
+import me.bickositieff.raspio.R
+
+class PlaylistFragment : Fragment() {
+
+    private lateinit var playlistViewModel: PlaylistViewModel
```
Comment (profiluefter) [inline app/app/src/main/java/me/bickositieff/raspio/ui/playlist/PlaylistFragment.kt:15]: Could have been initialized using `by viewModel()` (see https://stackoverflow.com/q/56748334/8719698) but results in the same

File: app/app/src/main/res/navigation/mobile_navigation.xml [hunk 3]
```diff
@@ -7,9 +7,9 @@
 
     <fragment
             android:id="@+id/navigation_home"
-            android:name="me.bickositieff.raspio.ui.home.HomeFragment"
+            android:name="me.bickositieff.raspio.ui.playlist.PlaylistFragment"
             android:label="@string/title_home"
```
Comment (profiluefter) [inline app/app/src/main/res/navigation/mobile_navigation.xml]: This should probably be changed to playlist or something
```suggestion
            android:label="@string/title_playlist"
```

File: app/app/src/main/res/values/strings.xml [hunk 4]
```diff
@@ -3,4 +3,5 @@
     <string name="title_home">Home</string>
```
Comment (profiluefter) [inline app/app/src/main/res/values/strings.xml]:
```suggestion
    <string name="title_playlist">Playlist</string>
```

**Review by profiluefter — CHANGES_REQUESTED** [review]
Fragment label should be changed

**Review by profiluefter — APPROVED** [review]

**Review by oagna — APPROVED** [review]

**Pull request merged**